roles/vendor/redis/tasks/os_configuration.yml

```diff
@@ -13,49 +13,40 @@
     create_home: no
     shell: /bin/false
 
-- name: Create redis cluster directories
-  file:
-    path: '{{ item }}'
-    state: directory
-    owner: redis
-    group: redis
-    mode: 0755
-    recurse: yes
-  loop:
-    - /etc/redis_cluster/redis_7000
-    - /etc/redis_cluster/redis_7001
-    - /var/lib/redis_cluster/redis_7000
-    - /var/lib/redis_cluster/redis_7001
-    - /var/log/redis_cluster
-    - /var/run/redis_cluster
-  when: redis_cluster == true
+# - name: Create redis cluster directories
+#   file:
+#     path: '{{ item }}'
+#     state: directory
+#     owner: redis
+#     group: redis
+#     mode: 0755
+#     recurse: yes
+#   loop:
+#     - /etc/redis_cluster/redis_7000
+#     - /etc/redis_cluster/redis_7001
+#     - /var/lib/redis_cluster/redis_7000
+#     - /var/lib/redis_cluster/redis_7001
+#     - /var/log/redis_cluster
+#     - /var/run/redis_cluster
+#   when: redis_cluster == true
```

Comment:
With GIT, there's no need to leave commented code lingering. Please remove this.